core/src/test/java/kafka/log/remote/RemoteLogManagerTest.java

```diff
@@ -269,15 +269,38 @@ void testStartup() {
     void testCopyLogSegmentsToRemoteShouldCopyExpectedLogSegment() throws Exception {
         long oldSegmentStartOffset = 0L;
         long nextSegmentStartOffset = 150L;
-        long oldSegmentEndOffset = nextSegmentStartOffset - 1;
+        long lso = 250L;
+        long leo = 300L;
+        assertCopyExpectedLogSegmentsToRemote(oldSegmentStartOffset, nextSegmentStartOffset, lso, leo);
+    }
+
+    /**
+     * The following values will be equal when the active segment gets rotated to passive when there are no new messages:
```

Comment:
nit: The following values will be equal when the active segment gets rotated to passive [and] there are no new messages: